components/toast.css

```diff
@@ -0,0 +1,39 @@
+.toast {
+  visibility: hidden;
+  min-width: 250px;
+  background-color: var(--info);
+  color: #fff;
+  text-align: center;
+  border-radius: var(--size-sm);
+  position: fixed;
+  z-index: 1;
+  left: 50%;
+  bottom: var(--size-xl);
+}
+.toast.show {
+  visibility: visible;
+  -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
```

Comment:
You don't need to add a browser vendor prefix until you are trying to support a very old browser version

Reply:
ok, thanks for pointing it out